docs/source/jit_language_reference.rst

```diff
@@ -346,6 +348,74 @@ like any other TorchScript type:
     print(sum_pair(p))
 
 
+.. _TorchScript Enum:
+.. _TorchScript Enums:
+.. _torchscript-enums:
+
+TorchScript Enums
+^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+    TorchScript enum support is experimental..
+
+Python enums can be used in TorchScript without any extra annotation or code:
```

Comment:
Personally I would cut down on this section a bit - it's pretty long for what is an easy & simple feature (enums).   see how short the named tuple section is

Reply:
What do you think I should cut out? Most of it is code samples.